File: vertx-lang-php-jphp/src/main/php/io/vertx/php/core/http/ServerWebSocket.php
<?php 
namespace io\vertx\php\core\http;
use io\vertx\php\core\MultiMap;
use io\vertx\php\core\buffer\Buffer;
use io\vertx\php\core\net\SocketAddress;
class ServerWebSocket
{
    
    private function __construct()
    {

    }
    /**
     * ()
     * @return Void
     * methodSize : 1
     */
    public function accept(/*0*/)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function binaryHandlerID(/*0*/)
    {

    }
    /**
     * (callable handler)
     * @return WebSocketBase
     * methodSize : 1
     */
    public function binaryMessageHandler(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return Void
     * methodSize : 1
     */
    public function close(/*0*/)
    {

    }
    /**
     * (callable handler)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function closeHandler(/*1*/$arg0)
    {

    }
    /**
     * (callable handler)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function drainHandler(/*1*/$arg0)
    {

    }
    /**
     * (Buffer t)
     * @return Void
     * ()
     * @return Void
     * methodSize : 2
     */
    public function end(/*1*/$arg0)
    {

    }
    /**
     * (callable endHandler)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function endHandler(/*1*/$arg0)
    {

    }
    /**
     * (callable handler)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function exceptionHandler(/*1*/$arg0)
    {

    }
    /**
     * (callable handler)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function frameHandler(/*1*/$arg0)
    {

    }
    /**
     * (callable handler)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function handler(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return MultiMap
     * methodSize : 1
     */
    public function headers(/*0*/)
    {

    }
    /**
     * ()
     * @return bool
     * methodSize : 1
     */
    public function isSsl(/*0*/)
    {

    }
    /**
     * ()
     * @return SocketAddress
     * methodSize : 1
     */
    public function localAddress(/*0*/)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function path(/*0*/)
    {

    }
    /**
     * ()
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function pause(/*0*/)
    {

    }
    /**
     * (callable handler)
     * @return WebSocketBase
     * methodSize : 1
     */
    public function pongHandler(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function query(/*0*/)
    {

    }
    /**
     * ()
     * @return Void
     * (int status)
     * @return Void
     * methodSize : 2
     */
    public function reject(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return SocketAddress
     * methodSize : 1
     */
    public function remoteAddress(/*0*/)
    {

    }
    /**
     * ()
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function resume(/*0*/)
    {

    }
    /**
     * (int maxSize)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function setWriteQueueMaxSize(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function subProtocol(/*0*/)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function textHandlerID(/*0*/)
    {

    }
    /**
     * (callable handler)
     * @return WebSocketBase
     * methodSize : 1
     */
    public function textMessageHandler(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return string
     * methodSize : 1
     */
    public function uri(/*0*/)
    {

    }
    /**
     * (Buffer data)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function write(/*1*/$arg0)
    {

    }
    /**
     * (Buffer data)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function writeBinaryMessage(/*1*/$arg0)
    {

    }
    /**
     * (Buffer data)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function writeFinalBinaryFrame(/*1*/$arg0)
    {

    }
    /**
     * (string text)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function writeFinalTextFrame(/*1*/$arg0)
    {

    }
    /**
     * (WebSocketFrame frame)
     * @return ServerWebSocket
     * methodSize : 1
     */
    public function writeFrame(/*1*/$arg0)
    {

    }
    /**
     * (Buffer data)
     * @return WebSocketBase
     * methodSize : 1
     */
    public function writePing(/*1*/$arg0)
    {

    }
    /**
     * (Buffer data)
     * @return WebSocketBase
     * methodSize : 1
     */
    public function writePong(/*1*/$arg0)
    {

    }
    /**
     * ()
     * @return bool
     * methodSize : 1
     */
    public function writeQueueFull(/*0*/)
    {

    }
    /**
     * (string text)
     * @return WebSocketBase
     * methodSize : 1
     */
    public function writeTextMessage(/*1*/$arg0)
    {

    }
}

File: vertx-lang-php-jphp/src/main/api/io/vertx/php/core/http/ServerWebSocket.java
package io.vertx.php.core.http;
import php.runtime.annotation.Reflection.Name;
import php.runtime.annotation.Reflection.Namespace;
import php.runtime.annotation.Reflection.Signature;
import io.vertx.lang.jphp.IWrapper;
import io.vertx.php.core.http.WebSocketBase;
import io.vertx.php.core.buffer.Buffer;
import io.vertx.php.core.MultiMap;
import io.vertx.php.core.http.WebSocketFrame;
import io.vertx.php.core.net.SocketAddress;
@Name("ServerWebSocket")
@Namespace("io\\vertx\\php\\core\\http")
public interface ServerWebSocket<S extends io.vertx.core.http.ServerWebSocket> extends IWrapper<S>{
    
    @Signature
    default void end(Environment __env__, Memory... args) {
        this.getWrappedObject().end()
    }

    @Signature
    default void end(Environment __env__, Memory... args) {
        this.getWrappedObject().end()
    }
    
    @Signature
    default Memory writeQueueFull(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().writeQueueFull()
    }
    
    @Signature
    default Memory binaryHandlerID(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().binaryHandlerID()
    }
    
    @Signature
    default Memory textHandlerID(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().textHandlerID()
    }
    
    @Signature
    default Memory subProtocol(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().subProtocol()
    }

    @Signature
    default Memory writeTextMessage(Environment __env__, Memory... args) {
        this.getWrappedObject().writeTextMessage(text)
    }

    @Signature
    default Memory writePing(Environment __env__, Memory... args) {
        this.getWrappedObject().writePing(data)
    }

    @Signature
    default Memory writePong(Environment __env__, Memory... args) {
        this.getWrappedObject().writePong(data)
    }

    @Signature
    default Memory textMessageHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().textMessageHandler(handler)
    }

    @Signature
    default Memory binaryMessageHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().binaryMessageHandler(handler)
    }

    @Signature
    default Memory pongHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().pongHandler(handler)
    }
    
    @Signature
    default void close(Environment __env__, Memory... args) {
        this.getWrappedObject().close()
    }

    @Signature
    default void close(Environment __env__, Memory... args) {
        this.getWrappedObject().close()
    }
    
    @Signature
    default Memory remoteAddress(Environment __env__, Memory... args) {
        return Utils.convReturnVertxGen(__env__, SocketAddressImpl::new, this.getWrappedObject().remoteAddress()
    }
    
    @Signature
    default Memory localAddress(Environment __env__, Memory... args) {
        return Utils.convReturnVertxGen(__env__, SocketAddressImpl::new, this.getWrappedObject().localAddress()
    }
    
    @Signature
    default Memory isSsl(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().isSsl()
    }

    @Signature
    default Memory exceptionHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().exceptionHandler(handler)
    }

    @Signature
    default Memory handler(Environment __env__, Memory... args) {
        this.getWrappedObject().handler(handler)
    }
    
    @Signature
    default Memory pause(Environment __env__, Memory... args) {
        this.getWrappedObject().pause()
    }
    
    @Signature
    default Memory resume(Environment __env__, Memory... args) {
        this.getWrappedObject().resume()
    }

    @Signature
    default Memory endHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().endHandler(endHandler)
    }

    @Signature
    default Memory write(Environment __env__, Memory... args) {
        this.getWrappedObject().write(data)
    }

    @Signature
    default Memory setWriteQueueMaxSize(Environment __env__, Memory... args) {
        this.getWrappedObject().setWriteQueueMaxSize(maxSize)
    }

    @Signature
    default Memory drainHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().drainHandler(handler)
    }

    @Signature
    default Memory writeFrame(Environment __env__, Memory... args) {
        this.getWrappedObject().writeFrame(frame)
    }

    @Signature
    default Memory writeFinalTextFrame(Environment __env__, Memory... args) {
        this.getWrappedObject().writeFinalTextFrame(text)
    }

    @Signature
    default Memory writeFinalBinaryFrame(Environment __env__, Memory... args) {
        this.getWrappedObject().writeFinalBinaryFrame(data)
    }

    @Signature
    default Memory writeBinaryMessage(Environment __env__, Memory... args) {
        this.getWrappedObject().writeBinaryMessage(data)
    }

    @Signature
    default Memory closeHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().closeHandler(handler)
    }

    @Signature
    default Memory frameHandler(Environment __env__, Memory... args) {
        this.getWrappedObject().frameHandler(handler)
    }
    
    @Signature
    default Memory uri(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().uri()
    }
    
    @Signature
    default Memory path(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().path()
    }
    
    @Signature
    default Memory query(Environment __env__, Memory... args) {
        return Utils.convReturnUnknow(__env__, this.getWrappedObject().query()
    }
    
    @Signature
    default Memory headers(Environment __env__, Memory... args) {
        return Utils.convReturnVertxGen(__env__, MultiMapImpl::new, this.getWrappedObject().headers()
    }
    
    @Signature
    default void accept(Environment __env__, Memory... args) {
        this.getWrappedObject().accept()
    }
    
    @Signature
    default void reject(Environment __env__, Memory... args) {
        this.getWrappedObject().reject()
    }

    @Signature
    default void reject(Environment __env__, Memory... args) {
        this.getWrappedObject().reject()
    }

}
